/tmp/redash/bin/release_manager.py:65:20: DUO123 use of "verify=False" is insecure in "requests" module
/tmp/redash/redash/models/users.py:176:21: DUO130 insecure use of "hashlib" module
/tmp/redash/redash/models/users.py:252:20: DUO130 insecure use of "hashlib" module
/tmp/redash/redash/query_runner/python.py:296:13: DUO105 use of "exec" is insecure
/tmp/redash/redash/query_runner/salesforce.py:150:17: DUO138 catastrophic "re" usage - denial-of-service possible
/tmp/redash/redash/query_runner/script.py:20:14: DUO116 use of "shell=True" is insecure in "subprocess" module
/tmp/redash/redash/tasks/schedule.py:43:12: DUO130 insecure use of "hashlib" module
/tmp/redash/redash/utils/__init__.py:61:12: DUO130 insecure use of "hashlib" module
